astro.config.mjs

```diff
@@ -23,6 +24,18 @@ export default defineConfig({
 			themes: [houston],
 			frames: false,
 		}),
+		icon({
+			svgoOptions: {
+				plugins: [
+					{ name: 'preset-default' },
+					{
+						name: 'prefixIds',
+						// Ensure IDs used in SVGs are unique to avoid clashes between inline SVGs.
```

Comment:
Ugh, good catch. Will fix this upstream!

Reply:
There is this open issue I found when I hit this: https://github.com/natemoo-re/astro-icon/issues/127